static/app/views/issueDetails/streamline/eventGraph.tsx

```diff
@@ -68,348 +94,434 @@ function createSeriesAndCount(stats: EventsStats) {
-    if (!unfilteredGroupStats?.['count_unique(user)']) {
-      return {series: []};
-    }
-    return createSeriesAndCount(unfilteredGroupStats['count_unique(user)']);
-  }, [unfilteredGroupStats]);
-  const userSeries = useMemo(() => {
-    if (!groupStats['count_unique(user)']) {
-      return [];
-    }
+    const {series: unfilteredEventSeries} = useMemo(() => {
+      if (!unfilteredGroupStats?.['count()']) {
+        return {series: []};
+      }
 
-    return createSeriesAndCount(groupStats['count_unique(user)']).series;
-  }, [groupStats]);
-
-  const chartZoomProps = useChartZoom({
-    saveOnZoom: true,
-  });
-
-  const currentEventSeries = useCurrentEventMarklineSeries({
-    event,
-    group,
-  });
-  const releaseSeries = useReleaseMarkLineSeries({group});
-  const flagSeries = useFlagSeries({
-    query: {
-      start: eventView.start,
-      end: eventView.end,
-      statsPeriod: eventView.statsPeriod,
-    },
-    event,
-  });
+      return createSeriesAndCount(unfilteredGroupStats['count()']);
+    }, [unfilteredGroupStats]);
+    const {series: unfilteredUserSeries} = useMemo(() => {
+      if (!unfilteredGroupStats?.['count_unique(user)']) {
+        return {series: []};
+      }
+      return createSeriesAndCount(unfilteredGroupStats['count_unique(user)']);
+    }, [unfilteredGroupStats]);
+    const userSeries = useMemo(() => {
+      if (!groupStats['count_unique(user)']) {
+        return [];
+      }
+
+      return createSeriesAndCount(groupStats['count_unique(user)']).series;
+    }, [groupStats]);
+
+    const chartZoomProps = useChartZoom({
+      saveOnZoom: true,
+    });
+
+    const currentEventSeries = useCurrentEventMarklineSeries({
+      event,
+      group,
+    });
+
+    const {releases = []} = useReleaseStats(
+      {
+        projects: eventView.project,
+        environments: eventView.environment,
+        datetime: {
+          start: eventView.start,
+          end: eventView.end,
+          period: eventView.statsPeriod,
+        },
+      },
+      {
+        staleTime: 0,
+      }
+    );
 
-  const series = useMemo((): BarChartSeries[] => {
-    const seriesData: BarChartSeries[] = [];
-    const translucentGray300 = Color(theme.gray300).alpha(0.5).string();
-    const lightGray300 = Color(theme.gray300).alpha(0.2).string();
+    const releaseSeries = useReleaseMarkLineSeries({
+      group,
+      releases: hasReleaseBubblesSeries && showReleasesAs !== 'line' ? [] : releases,
+    });
+
+    const {
+      createReleaseBubbleHighlighter,
+      releaseBubbleEventHandlers,
+      releaseBubbleSeries,
+      releaseBubbleXAxis,
+      releaseBubbleGrid,
+    } = useReleaseBubbles({
+      chartRenderer: ({chartRef}) => {
+        return (
+          <EventGraph
+            ref={chartRef}
+            group={group}
+            event={event}
+            showSummary={false}
+            showReleasesAs="line"
+            disableZoomNavigation
```

Comment:
This is the chart that gets rendered in the drawer, these are the custom options that differentiate it from the main chart.